airbyte-webapp/src/views/Connector/ConnectorCard/ConnectorCard.tsx

```diff
@@ -73,6 +73,8 @@ export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEdi
         trackTestConnectorSuccess(connector);
       } catch (e) {
         trackTestConnectorFailure(connector);
+        const connectorFailureCount = Number(localStorage.getItem("connector_failure_count"));
+        localStorage.setItem("connector_failure_count", (connectorFailureCount + 1).toString());
```

Comment:
This will store this just arbitrarily for any connector and count up the count for any connector. I don't think it makes much sense having a counter that counts no matter which connector fails just up, and this should likely be scoped to each specific connector.

Also if we use string constants like this, they should be properly extracted into a `const` variable and not repeated in different places.

Very generally: we send an event here already for the failure. Can't we actually leverage those events to track what we're trying to achieve instead adding a new hacky localStorage variable here. We e.g. have functions in Segment that could potentially track this? (cc @letiescanciano)

Also if we keep this code for an experiment, please make sure to highlight it in a comment which experiment it belongs to, so you'll remember to clean it up, once we remove the experiment.